bsp/boards/openmote-b/board.c

```diff
@@ -18,7 +18,9 @@
 #include <source/gptimer.h>
 #include <source/sys_ctrl.h>
 
+#include "opentimers.h"
```

Comment:
No. opentimers is at `driver/` level. You cannot include a driver-level module in a BSP module.